### ISSUE TITLE
app_state should be added to the login & register method

### ISSUE THREAD

**Issue by ClintDavis** [luukhaijes/kinde-angular#5]
As per https://kinde.com/docs/developer-tools/javascript-sdk/#persisting-application-state

```
type AuthOptions = {
  org_code?: string;
  app_state?: object;
  authUrlParams?: object;
};
```

Should pass through the login and register methods, allowing for things like redirect to be managed.

**Comment by luukhaijes** [comment]
Hi @ClintDavis, thank you for your feedback. I'm currently working on the issue. Please note that this package utilizes the TypeScript SDK, which uses slightly different props from those used by the JavaScript SDK.

**Comment by luukhaijes** [comment]
Fixed by 4e332a7
